modules/env_parameters/functions/ENV/fn_ENV_preInit.sqf

```diff
@@ -0,0 +1,53 @@
+#include "script_component.hpp"
+#include "..\..\settings.sqf"
+
+if (isMultiplayer) then {
```

Comment:
if you just want to exit the script on a condition use exitwith

Reply:
Fixed.